### PR TITLE
☠️ Kensur PR

### PR DESCRIPTION
**Pull request by Phyco-Ninja**
Contains Spam, Ban all, and Kick all

### CODE REVIEW

**Comment by Phyco-Ninja** [comment]
@pep8speaks bruh u comedy me??  First u said remove trailing white spaces and naw u want blank lines :/ ,,,

File: plugins/spem.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        if replied.sticker:
+            to_spem = replied.sticker.file_id
+            try:
+                sc = int(message.input_str)
+            except ValueError as e:
+                await message.edit(e)
+                await message.reply_sticker(sticker="CAADAQADzAADiO9hRu2b2xyV4IbAFgQ")
+                return
+            await message.edit(f"Spamming {sc} Time")
+            for spem in range(sc):
```
Comment (rking32) [inline plugins/spem.py]: unused variable `spem` 🤔

Comment (Phyco-Ninja) [reply plugins/spem.py]: Idk 🤔 used underscore naw

**Comment by Phyco-Ninja** [comment]
🤔 Codacy why are u so bitchy bro??? 😐

File: plugins/spem.py (hunk 1 above)
Comment (rking32) [reply plugins/spem.py]: cool

**Comment by rking32** [comment]
yeah, what is that error :thinking:

**Comment by Phyco-Ninja** [comment]
Blame dev of Codacy 🤔🤔

**Pull request merged**